src/Billing/Utilities/PayPalIpnClient.cs

```diff
@@ -29,19 +29,16 @@ public async Task<bool> VerifyIpnAsync(string ipnBody)
             throw new ArgumentException("No IPN body.");
         }
 
-        var request = new HttpRequestMessage
-        {
-            Method = HttpMethod.Post,
-            RequestUri = _ipnUri
-        };
-        _httpClient.DefaultRequestHeaders.Add("User-Agent", "CSharp-IPN-VerificationScript");
+        var request = new HttpRequestMessage { Method = HttpMethod.Post, RequestUri = _ipnUri };
         var cmdIpnBody = string.Concat("cmd=_notify-validate&", ipnBody);
```

Comment:
Since we don't have it in DataDog, do you think it's worth logging the `ipnBody` as well? Not sure if there's any sensitive information in it.

Reply:
Sure thing, one moment